src/assets/js/admin/new-optin-campaign.js

```diff
@@ -10,6 +10,8 @@
 
             if (!optin_campaign_title_obj.val()) {
                 optin_campaign_title_obj.addClass('mailoptin-input-error');
+                alert('Title of the Optin Campaign is required');
```

Comment:
This string is not translatable. Go to wp-content/plugins/mailoptin/Core/src/RegisterScripts.php Line 237. Add the string to the js object and then call the string via `mailoptin_globals.objectItemName`.

Read more on wp_localize_script here https://developer.wordpress.org/reference/functions/wp_localize_script/

Reply:
okay